cli/internal/cloudcmd/create.go

```diff
@@ -54,7 +59,7 @@ func NewCreator(out io.Writer) *Creator {
 }
 
 // Create creates the handed amount of instances and all the needed resources.
-func (c *Creator) Create(ctx context.Context, provider cloudprovider.Provider, config *config.Config, insType string, controlPlaneCount, workerCount int,
+func (c *Creator) Create(ctx context.Context, provider cloudprovider.Provider, config *config.Config, azurePolicyPatcher PolicyPatcher, insType string, controlPlaneCount, workerCount int,
```

Comment:
create the PolicyPatcher in NewCreator instead of creating it in the cmd package and passing it